weasyprint/__main__.py

```diff
@@ -179,6 +185,10 @@ def main(argv=None, stdout=None, stdin=None, HTML=HTML):
                 optimize_size.add(arg)
     del args.optimize_size
 
+    url_fetcher = default_url_fetcher
+    if args.timeout:
```

Comment:
Adding `is not None` would allow users to set 0 if they want (I don’t know if it’s useful, but I have the intuition that some users will find this useful for some reason!)

Reply:
Makes sense. Will fix it.